projects/ui-components/lf-user-feedback/feedback-image-upload/feedback-image-upload.component.ts

```diff
@@ -9,12 +9,11 @@ import { AppLocalizationService } from '@laserfiche/lf-ui-components/internal-sh
 })
 export class FeedbackImageUploadComponent {
   @Output() imageUploadError: EventEmitter<string> = new EventEmitter<string>();
-  @Output() feedbackImageBase64: EventEmitter<string> = new EventEmitter<string>();
+  @Output() feedbackImageBase64: EventEmitter<string | undefined> = new EventEmitter<string | undefined>();
 
   @ViewChild('uploadFile') inputFile?: ElementRef<HTMLInputElement>;
 
-  imageUploaded?: File;
-  rawImageBase64: string = '';
+  imageUploaded?: {name: string; rawBase64: string} | undefined;
```

Comment:
does the question mark make the  | undefined typing redundant? Usually I just see one or the other.